game/core/src/me/lihq/game/screen/elements/OrthogonalTiledMapRendererWithSprite.java

```diff
@@ -51,6 +71,18 @@ public void render()
     {
         beginRender();
 
+        /*
+        Convert the sprites list into a list of AbstractPerson to be sorted
+         */
+        List<AbstractPerson> people = new ArrayList<AbstractPerson>();
+
+        for (Sprite sprite : sprites)
+        {
+            people.add((AbstractPerson) sprite);
```

Comment:
if we only end up casting the sprites into abstract people then we might as well just take abstract people? as they have a draw method we can call in line 97